### ISSUE TITLE
Feature Request: Order Tabs

### ISSUE THREAD

**Issue by lightsourced** [MSchmoecker/HammerTime#9]
lol it's me again! I was wondering if it would be possible to order the tabs that are added to the build hammer via the config. Would be pretty neat, there are some modded items I use more than others and would like those to appear first when cycling through!

**Comment by MSchmoecker** [comment]
Yes this sounds possible. The custom tabs are implemented inside of Jotunn, so we have to discuss how to do this best as those tabs are also used by other mods. Not a big blocker as I'm one of the devs there but it may take some time as we have to come up with a good solution.

Additionally, it sounds like the overall problem is that common used pieces are split in many tabs, which only comes more apprent with more mods. I was already thinking about adding a favourite category to the hammer where any piece can be moved to. The idea isn't fleshed out yet but may be useful. Anyway, I think that would fit into a different mod as it only compliments the selection of pieces and could also be used without HammerTime.

**Comment by lightsourced** [comment]
It could be that it collects all the tabs on run time as you do with hammer
time and then it prints them out on the config and then there is a dropdown
that allows you to select 1-10 or whatever how many extra tabs there
actually are. Stardew valley integrated mine carts mod does something like
that to organize the position of mine cart locations in its in-game
selection ui.

https://www.nexusmods.com/stardewvalley/mods/11881

Not sure if this is helpful or not :)
